rigex-front-end/src/components/rigScheduler/Header.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import './headerStyle.scss';
+
+import Mockup from '../../assets/Rig-Scheduler-Mockup.png';
```

Comment:
Let's use faker for our images. You shouldn't need to use any png files

Reply:
fixed - I used image.business since there is no logo option in the non-ruby version of faker.